lib/defaults-utils.js

```diff
@@ -25,17 +25,17 @@ export function toXmlArg (value, serialize = true) {
       const keyEl = xmlDoc.createElement('key');
       const keyTextEl = xmlDoc.createTextNode(subKey);
       keyEl.appendChild(keyTextEl);
-      xmlDoc.documentElement.appendChild(keyEl);
+      xmlDoc.documentElement?.appendChild(keyEl);
```

Comment:
should we instead assert that xmlDoc.documentElement is not null?

Reply:
Used type import for them since these parsing values are pretty simple. Should not be null basically